docs/usage/help.rst.inc

```diff
@@ -8,16 +8,18 @@ borg help patterns
 
 File patterns support these styles: fnmatch, shell, regular expressions,
 path prefixes and path full-matches. By default, fnmatch is used for
-`--exclude` patterns and shell-style is used for `--pattern`. If followed
-by a colon (':') the first two characters of a pattern are used as a
+`--exclude` patterns and shell-style is used for the experimental `--pattern`
+option.
+
+If followed by a colon (':') the first two characters of a pattern are used as a
 style selector. Explicit style selection is necessary when a
 non-default style is desired or when the desired pattern starts with
 two alphanumeric characters followed by a colon (i.e. `aa:something/*`).
 
 `Fnmatch <https://docs.python.org/3/library/fnmatch.html>`_, selector `fm:`
 
     This is the default style for --exclude and --exclude-from.
-    These patterns use a variant of shell pattern syntax, with '*' matching
+    These patterns use a variant of shell pattern syntax, with '\*' matching
```

Comment:
`*italic*`. Interestingly enough the rST parser does not pick this one up; I suspect it has a rule that the smallest range has in-line markup applied, i.e. in-line markup seems to be not-greedy in rST. Note that rST does not allow nested in-line markup. The escape is correctly processed, of course.

(Keep in mind: nanorst doesn't have to work for arbitrary stuff, it just needs to work for the rst used in the help texts)